devices/cpu/registers.h

```diff
@@ -25,6 +25,8 @@
 #include <stdbool.h>
 #include "../../main.h"
 
+enum {CallTracer_MaxCallDepth = 128};
```

Comment:
spaces ?

Reply:
Why enum? Maybe define, const or constexpr?

Reply:
"C Style"

`const int` in C is not what it is in `C++` - it still introduces "external symbol" and putting it in .h results in "duplicated symbols" link error. `static const int` in .h looks bad. Enum is still better than define :) (yep, C sucks)

Reply:
Spaces, ok